tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -20,6 +22,7 @@
 
 __metaclass__ = type
 
+DEFAULT_HLQ = "USER.PRIVATE"
```

Comment:
this is also a hard coded content right? Maybe we can include in the same data set name generator a function that generates random valid HLQs.